tests/integration_tests/test_external_data.py

```diff
@@ -16,6 +19,20 @@ def test_external_simple(test_client: Client):
     assert result[0][0] == '12 Angry Men'
 
 
+def test_external_arrow(test_client: Client):
+    if not arrow:
+        pytest.skip('PyArrow package not available')
```

Comment:
It might make sense to use `pytest.importorskip("pyarrow")` to avoid the boilerplate of having to write an error message.